src/core/thread/fiber.d

```diff
@@ -548,8 +549,8 @@ class Fiber
      * In:
      *  fn must not be null.
      */
-    this( void function() fn, size_t sz = PAGESIZE * defaultStackPages,
-          size_t guardPageSize = PAGESIZE ) nothrow
+    this( void function() fn, size_t sz = pageSize * defaultStackPages,
```

Comment:
Does using a fixed number of memory pages make more sense here than a fixed number of bytes?

Reply:
Perhaps, but that would be an API change.

Reply:
I meant the default values. Having thought more I believe the requested stack size should be some fixed number of bytes. Having 2MB memory pages shouldn't make fibers _default_ to having 8MB or 16MB of stack (those would be thick fibers indeed). Thanks to other code in this PR the allocation is rounded up to a full page anyway.

Making `guardPageSize` default to the actual page size is good, since the two most sensible values are the real page size or 0.